src/main/java/com/ldtteam/blockui/views/ScrollingListContainer.java

```diff
@@ -24,15 +24,36 @@ public class ScrollingListContainer extends ScrollingContainer
      */
     public void refreshElementPanes(final ScrollingList.DataProvider dataProvider, final PaneParams listNodeParams, final int height, final int childSpacing)
     {
+        int currentYpos = 0;
+
+        final Pane template = Loader.createFromPaneParams(listNodeParams, null);
```

Comment:
Is this a great idea to constantly load this?

Reply:
I reworked this to only load when there's any items that have to be rendered, better?

Reply:
better, but I still dont fully get why this is needed

Reply:
I need to know the dimensions of the template element so I can set the width/height of the "event".

At the time that I'm getting the new width, the actual elements have not been made yet, so I need this temporary element to quickly obtain the size, however this is only a one time thing that I do at the start.

Reply:
Yeah, but refreshElementPanes is, as far as I remember, called quite often.

Reply:
yeah, sourced from client tick, mby load once at pane ctor and just save the size (cuz afaik you dont need more than that)?